common/src/main/java/com/wynntils/features/ui/WynntilsContentBookFeature.java

```diff
@@ -103,100 +92,56 @@ public class WynntilsContentBookFeature extends Feature {
             () -> WynntilsMenuScreenBase.openBook(WynntilsGuidesListScreen.create()));
 
     @Persisted
-    public final Config<Boolean> replaceWynncraftContentBook = new Config<>(true);
-
-    @Persisted
-    public final Config<InitialPage> initialPage = new Config<>(InitialPage.USER_PROFILE);
-
-    @Persisted
-    public final Config<Boolean> showContentBookLoadingUpdates = new Config<>(true);
-
-    @Persisted
-    public final Config<Boolean> displayOverallProgress = new Config<>(true);
-
-    @Persisted
-    public final Config<Boolean> cancelAllQueriesOnScreenClose = new Config<>(true);
-
-    // WynntilsQuestBookScreen storages
-    @Persisted
-    public final Storage<Boolean> questsSelected = new Storage<>(true);
-
-    @Persisted
-    public final Storage<Boolean> miniQuestsSelected = new Storage<>(false);
-
-    // WynntilsDiscoveriesScreen storages
-    // Note: These are intentionally not more advanced types, like Map<DiscoveryType, Boolean>,
-    // to allow easier changes in the future (when more discovery types are added).
-    // TODO: Change this when storage upfixers are implemented
-    @Persisted
-    public final Storage<Boolean> secretsSelected = new Storage<>(true);
-
-    @Persisted
-    public final Storage<Boolean> undiscoveredSecretsSelected = new Storage<>(false);
-
-    @Persisted
-    public final Storage<Boolean> worldSelected = new Storage<>(true);
-
-    @Persisted
-    public final Storage<Boolean> undiscoveredWorldSelected = new Storage<>(false);
+    private final Config<ShiftBehavior> shiftBehaviorConfig = new Config<>(ShiftBehavior.DISABLED_IF_SHIFT_HELD);
 
-    @Persisted
-    public final Storage<Boolean> territorySelected = new Storage<>(true);
-
-    @Persisted
-    public final Storage<Boolean> undiscoveredTerritorySelected = new Storage<>(false);
+    private boolean shiftClickedBookItem = false;
 
     @SubscribeEvent
     public void onUseItem(UseItemEvent event) {
-        if (McUtils.player().isShiftKeyDown() || !replaceWynncraftContentBook.get()) return;
-
-        tryCancelQuestBookOpen(event);
+        shiftClickedBookItem = McUtils.player().isShiftKeyDown();
     }
 
     @SubscribeEvent
     public void onUseItemOn(PlayerInteractEvent.RightClickBlock event) {
-        if (McUtils.player().isShiftKeyDown() || !replaceWynncraftContentBook.get()) return;
-
-        tryCancelQuestBookOpen(event);
+        shiftClickedBookItem = McUtils.player().isShiftKeyDown();
     }
 
     @SubscribeEvent
     public void onInteract(PlayerInteractEvent.Interact event) {
-        if (McUtils.player().isShiftKeyDown() || !replaceWynncraftContentBook.get()) return;
-
-        tryCancelQuestBookOpen(event);
+        shiftClickedBookItem = McUtils.player().isShiftKeyDown();
     }
 
-    private void tryCancelQuestBookOpen(ICancellableEvent event) {
-        // Tutorial safeguard, don't replace the content book if the player hasn't completed the tutorial.
```

Comment:
This new book works well in the tutorial? I had this vague memory that the tutorial quest book was slightly different in that it tried to highlight different features, but maybe I'm just mis-remembering.